src/styles/blocks/_stars.scss

```diff
@@ -0,0 +1,14 @@
+.stars {
+  display: flex;
+
+  &__star {
+    margin-right: 4px;
+    width: 16px;
+    height: 16px;
+    background: url("../images/star.svg") no-repeat center;
+
+    .stars--4 &:nth-child(-n+4) {
```

Comment:
styles for stars should be for any amount of stars
and pay attention on this point from [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)
![Screenshot from 2023-05-11 13-47-25](https://github.com/mate-academy/layout_catalog/assets/33029864/f4adae40-ab72-44b9-9f6a-288a852316b7)